.eslintrc.js

```diff
@@ -95,6 +95,9 @@ module.exports = {
       rules: {
         ...tsRuleOverrides,
         ...jestRuleOverrides
+      },
+      parserOptions: {
+        project: './tsconfig.json'
```

Comment:
Required for the new versions of typescript-eslint packages (I think this should always have been there but didn't cause errors before)